bin/acceptance-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+
+case $1 in
+    chrome)
+        BROWSER_NAME=chrome
+        PLATFORM="Windows 10"
+        ;;
+    firefox)
+        BROWSER_NAME=firefox
+        BROWSER_VERSION="57.0"
```

Comment:
Should this be using `latest`?

```suggestion
        BROWSER_VERSION=latest
```